webapp/package.json

```diff
@@ -62,6 +64,10 @@
     "awesome-typescript-loader": "^5.2.1",
     "babel-eslint": "^10.0.1",
     "babel-loader": "^8.0.6",
+    "babel-plugin-transform-decorators-legacy": "^1.3.5",
```

Comment:
@enicolasgomez is this a consequence of branching from development?
This should've been removed since not used anyamore